src/emulator/storage/cloudFunctions.ts

```diff
@@ -1,18 +1,24 @@
 import { EmulatorRegistry } from "../registry";
 import { EmulatorInfo, Emulators } from "../types";
-import * as request from "request";
 import { EmulatorLogger } from "../emulatorLogger";
 import { CloudStorageObjectMetadata, toSerializedDate } from "./metadata";
 import { Client } from "../../apiv2";
 
 type StorageCloudFunctionAction = "finalize" | "metadataUpdate" | "delete" | "archive";
+const STORAGE_V2_ACTION_MAP: Record<string, string> = {
+  finalize: "finalized",
+  metadataUpdate: "metadataUpdated",
+  delete: "deleted",
+  archive: "archived",
+};
 
 export class StorageCloudFunctions {
   private logger = EmulatorLogger.forEmulator(Emulators.STORAGE);
   private functionsEmulatorInfo?: EmulatorInfo;
   private multicastOrigin = "";
   private multicastPath = "";
   private enabled = false;
+  private client: Client = new Client({ urlPrefix: "" });
```

Comment:
Since we only initialize in the constructor when `functionsEmulator` is set, we need to have `client` be be initialized to some default value (can't be null/undefined) so this seemed like a reasonable place to set a default value